recipes/python-black

```diff
@@ -0,0 +1,3 @@
+(gsettings
```

Comment:
Should be `python-black`.

Reply:
oops, i need to work on my copy/paste skills :wink: fixed now